wordpress-mobile/WordPress-iOS
# Adds tracking for all of the site settings areas

Project: #17503

⚠️ Depends on: https://github.com/wordpress-mobile/WordPress-iOS/pull/17539
View only this PRs [changes here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17545/files/b1803c087a1f29c4a8d24116579f02ad999eb58c..87a15d908ab723ccfda37652e5abb3224a41f8e7)


### Description
Adds tracking for **every** option in Site Settings and its encompassing settings

### To test:
This affects every item on Site Settings that did not already have tracks

1. Launch the app
2. Tap on the My Site tab
3. Tap on Site Settings
4. Go through each setting on the main view, and the sub views and verify you see for each of the settings changed:
    - `🔵 Tracked: settings_did_change <field_name: FIELD_NAME, page: PAGE_NAME>`

Note: Areas with existing tracks were not changed, and tracking was not added to those areas

### Regression Notes
1. Potential unintended areas of impact
None, adding tracking.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
None.

### PR submission checklist:
